app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -15,17 +65,18 @@ export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
     .object()
     .nullable()
     .shape({
-      label: yup.string(),
-      value: yup.string(),
+      label: yup.string().oneOf(ASSIGNED_CONTACT_OPTIONS.map(tag => tag.label)),
+      value: yup.string().oneOf(ASSIGNED_CONTACT_OPTIONS.map(tag => tag.value)),
     })
     .required("Assigned Contact is required"),
-  tag: yup
-    .object()
+  tags: yup
+    .array(
+      yup.object().shape({
+        label: yup.string().oneOf(TAG_OPTIONS.map(tag => tag.label)),
+        value: yup.string().oneOf(TAG_OPTIONS.map(tag => tag.value)),
+      })
+    )
     .nullable()
```

Comment:
Refer above comment and update this validation schema accordingly.

Reply:
This is resolved in #18